message_ix_models/model/water/data/water_supply.py

```diff
@@ -400,8 +401,8 @@ def add_water_supply(context: "Context") -> dict[str, pd.DataFrame]:
                 make_df(
                     "input",
                     technology="extract_surfacewater",
-                    value=0.018835616,
-                    unit="-",
+                    value=0.018835616 / 1e3,
```

Comment:
also here, why some conversion factors are with name and other are not? you can also use the KM3_TO_MCM and divide by it, if that was the case. It does not have to be GWa_KM3_to_GWa_MCM